xa/java/department-spring-jpa/pom.xml

```diff
@@ -67,27 +67,32 @@
 		<dependency>
 			<groupId>com.oracle.database.jdbc</groupId>
 			<artifactId>ojdbc8</artifactId>
-			<version>21.3.0.0</version>
+			<version>21.9.0.0</version>
 		</dependency>
 		<dependency>
 			<groupId>com.oracle.database.jdbc</groupId>
 			<artifactId>ucp</artifactId>
-			<version>21.3.0.0</version>
+			<version>21.9.0.0</version>
 		</dependency>
 		<dependency>
 			<groupId>com.oracle.database.security</groupId>
 			<artifactId>oraclepki</artifactId>
-			<version>21.3.0.0</version>
+			<version>21.9.0.0</version>
 		</dependency>
 		<dependency>
 			<groupId>com.oracle.database.security</groupId>
 			<artifactId>osdt_core</artifactId>
-			<version>21.3.0.0</version>
+			<version>21.9.0.0</version>
 		</dependency>
 		<dependency>
 			<groupId>com.oracle.database.security</groupId>
 			<artifactId>osdt_cert</artifactId>
-			<version>21.3.0.0</version>
+			<version>21.9.0.0</version>
+		</dependency>
+		<dependency>
+			<groupId>javax.xml.bind</groupId>
+			<artifactId>jaxb-api</artifactId>
+			<version>2.3.0</version>
```

Comment:
Why do we need this dependency?  In spring boot we are using the Jakarta extension, if you need this then you most probably may need to add from groupId "jakarta.xml.bind"

Reply:
There was some Javax dependency issue in hibernate 6.3+. after resolving that I have removed it